feature-libs/product-configurator/rulebased/core/state/reducers/configurator.reducer.ts

```diff
@@ -453,19 +452,10 @@ function takeOverPricingChanges(
   state: Configurator.Configuration
 ): Configurator.Configuration {
   const content = { ...action.payload };
-  const priceSupplements = content.priceSupplements;
-  const groups =
-    priceSupplements && priceSupplements.length > 0
-      ? ConfiguratorStateUtils.mergeGroupsWithSupplements(
-          state.groups,
-          priceSupplements
-        )
-      : state.groups;
 
   const result = {
     ...state,
     ...content,
-    groups: groups,
```

Comment:
It's for sure a breaking change for customers that relied on the fact that those prices are attached as leafs of the tree.

I believe we should hide this breaking change behind a feature toggle.
That said, I'd need to think a bit on how to introduce `FeaturesConfigService` here as an injectable service - into the reducer 🤔 
IMHO we shold not apss a service via action's payload. Actions should not care about such things.
Maybe we could pass a service _somehow_ as an ambient context, using _some_ Ngrx API, maybe a meta reducer? I'd need to dig deeper and investigate. Any quick ideas from top of your head?